.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -411,19 +411,19 @@ try{
         "Host verification package 2004 RelWithDebInfo": { ACCHostVerificationPackageTest('20.04', 'RelWithDebInfo') },
         "Host verification package 1804 RelWithDebInfo": { ACCHostVerificationPackageTest('18.04', 'RelWithDebInfo') },
 
-        "ACC1804 clang-8 Release EEID Experimental LVI FULL Tests": { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'clang-8', 'Release', ['-DLVI_MITIGATION=ControlFlow', '-DLVI_MITIGATION_SKIP_TESTS=OFF', '-DWITH_EEID=ON']) },
+        "ACC1804 clang-10 Release EEID Experimental LVI FULL Tests": { ACCTest(AGENTS_LABELS["acc-ubuntu-18.04"], 'clang-10', 'Release', ['-DLVI_MITIGATION=ControlFlow', '-DLVI_MITIGATION_SKIP_TESTS=OFF', '-DWITH_EEID=ON']) },
```

Comment:
Suggest using `clang` on the stage name.

Reply:
If we change this to just clang it pushes the decision making of which clang version to use to this script: https://github.com/openenclave/openenclave-ci/blob/master/src/jenkins/common/Openenclave.groovy#L65

I think for visibility, and to make it easier to test out experimental changes (like updating to a new clang version) we would want to keep using `clang-<version>`.